app/assets/stylesheets/style.css

```diff
@@ -101,6 +101,10 @@ a .fa-white,
     color: white !important;
 }
 
+.fa-bars {
```

Comment:
Ah, actually this will affect /all/ uses of this class. Would it be possible to do this as an inline style change instead, in `/app/views/layouts/_header.html.erb`? Thank you!

Reply:
sure. i will do an inline style change for review. let me know if there is anything else that needs work. thanks for your time!